css-text/text-indent/text-indent-hanging-001.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en" >
+<head>
+<meta charset="utf-8">
+<title>CSS Text: text-indent: hanging</title>
+<meta name="flags" content="font">
+<meta name="assert" id="assert" content="text-indent: hanging makes all lines except the first line as indented">
+<link rel="help" href="https://drafts.csswg.org/css-text-3/#text-indent-property">
+<link rel="author" title="hiroshi takase" href="mailto:takase@est.co.jp">
+<link rel="stylesheet" href="../../fonts/metadata_styles.css"/>
+<style>
+    @font-face {
+        font-family: 'webfont';
+        src: url('../../../fonts/ahem-extra/AHEM_default.TTF') format('opentype');
+        font-weight: normal;
+        font-style: normal;
+    }
+    #test, #ref {
+        position:absolute; top:0;
+        font-family: webfont;
+    }
+    #test {
+        width: 20em;
+        color: green;
+    }
+    #ref {
+        width: 15em;
+        padding-left: 5em;
+        text-indent: -5em;

```

Comment:
I don't think you should use text-indent for styling the ref text. If it doesn't work, the test will fail because the ref file was incorrect.  Given that this is using a fixed-width font, where each glyph is 1em wide, perhaps it would be better to create the ref text with line breaks at the end of each line and 5 space characters at the beginning of each line except the first.  Also, you won't need to set the width to 15em or pad that way.